iree/compiler/Codegen/Common/IREEComprehensiveBufferizePass.cpp

```diff
@@ -80,10 +202,22 @@ class IREEComprehensiveBufferizePass
     registry.insert<arith::ArithmeticDialect, IREE::Util::UtilDialect,
                     linalg::LinalgDialect, memref::MemRefDialect,
                     scf::SCFDialect, StandardOpsDialect, tensor::TensorDialect,
-                    vector::VectorDialect, AffineDialect>();
+                    vector::VectorDialect, AffineDialect,
+                    IREE::Flow::FlowDialect>();
+
+    // Registers operations of other dialects.
     linalg::comprehensive_bufferize::
         registerBufferizableOpInterfaceExternalModels(registry);
+    linalg::comprehensive_bufferize::linalg_ext::
```

Comment:
Could you add a TODO here to move these registrations out of here. There might be a better place to add these.